### ISSUE TITLE
Include box_totals for http_1down/up

### ISSUE THREAD

**Issue by eriktar** [tohojo/flent#150]
I find the box_totals graph very useful in tcp_1up/down.  Would be nice to have it along in the http tests as well.

**Comment by tohojo** [comment]
Do you mean a box plot of just the bandwidth, or one that includes HTTP fetch times?

**Comment by eriktar** [comment]
The one with bandwidth and http fetch time.  If you look at the graphs
produced at my side, the box_combine (which is in there pr default) is a
bit hard to read.  But the box is more readable.

PLOTS['box_combine'] = {'description': 'Box combination of HTTP latency and
TCP bandwidth',
*                        'type': 'box',  # vs box_combine*
                        'axis_labels': ['Mean bandwidth (MBits/s)', 'Mean
download time (ms)'],
                        'dual_axes': True,
                        'series': [
                            {'data': 'TCP download',
                                    'label': 'TCP download',
                                    'combine_mode': 'meta:MEAN_VALUE'},
                            {'data': 'HTTP latency',
                                    'label': 'HTTP get (ms)',
                                    'combine_mode': 'mean',
                                    'axis': 2}]}

**Comment by tohojo** [comment]
Yeah, the box_combine plot is not terribly useful with a single test. It plots the distribution of averages over several test runs; so try it with multiple files.

I'll add the other type as well :)

**Comment by eriktar** [comment]
Perhaps I'm doing it wrong, but I still find box more readable.

**Comment by tohojo** [comment]
eriktar <notifications@github.com> writes:

> Perhaps I'm doing it wrong, but I still find box more readable.

Right, sure. Be aware that because box plots express percentiles, they
can be misleading for data sets that do not consist of independent
samples, such as timeseries data points from the same download.

I'm not saying they can't be useful; just be careful which conclusions
you draw. For instance, saying things like "25% of downloads were below
X Mbps" from a box plot would probably not be valid...
